src/contracts/implementations/arbitrable/MultipleArbitrableTransaction.js

```diff
@@ -0,0 +1,364 @@
+import multipleArbitrableTransactionArtifact from 'kleros-interaction/build/contracts/MultipleArbitrableTransaction'
+import _ from 'lodash'
+
+import * as contractConstants from '../../../constants/contract'
+import * as errorConstants from '../../../constants/error'
+import deployContractAsync from '../../../utils/deployContractAsync'
+
+import Arbitrable from './Arbitrable'
+
+/**
+ * Provides interaction with an Arbitrable Transaction contract deployed on the blockchain.
+ */
+class MultipleArbitrableTransaction extends Arbitrable {
+  /**
+   * Constructor ArbitrableTransaction.
+   * @param {object} web3Provider instance
+   * @param {string} contractAddress of the contract
+   */
+  constructor(web3Provider, contractAddress) {
+    super(web3Provider, multipleArbitrableTransactionArtifact, contractAddress)
+
+    this.arbitrableTransactionId = null
+  }
+
+  /**
+   * Deploy MultipleArbitrableTransaction.
+   * @param {object} account Ethereum account (default account[0])
+   * @param {object} web3Provider web3 provider object
+   * @returns {object} truffle-contract Object | err The deployed contract or an error
+   */
+  static deploy = async (account, web3Provider) => {
+    const contractDeployed = await deployContractAsync(
+      account,
+      0,
+      multipleArbitrableTransactionArtifact,
+      web3Provider
+    )
+
+    return contractDeployed
+  }
+
+  /**
+   * Create MultipleArbitrableTransaction.
+   * @param {object} account Ethereum account (default account[0])
+   * @param {string} arbitratorAddress The address of the arbitrator contract
+   * @param {object} seller Seller Ethereum account
+   * @param {number} value funds to be placed in contract
+   * @param {number} timeout Time (seconds) after which a party automatically loose a dispute. (default 3600)
+   * @param {bytes} arbitratorExtraData Extra data for the arbitrator. (default empty string)
+   * @param {string} metaEvidenceUri Uri meta-evidence. (default empty string)
+   * @returns {object} truffle-contract Object | err The deployed contract or an error
+   */
+  createArbitrableTransaction = async (
+    account = this._Web3Wrapper.getAccount(0),
```

Comment:
I have been taking all these defaults for account out. I think its better if the caller passes the correct account. Especially if its the first param they will have to do it anyways since there are no kwargs

Reply:
I don't understand what you say. Can you write the parameters that you want?